crates/egui-wgpu/src/winit.rs

```diff
@@ -668,6 +668,7 @@ impl Painter {
 
         {
             crate::profile_scope!("present");
+            // This is where vsync happens in DX12
```

Comment:
apologies I got that wrong at first, plz remove this line again :). The wait is always happening `get_current_texture`, but internally (!) what is waited on can be subtly different

See 
* https://github.com/gfx-rs/wgpu/issues/3283
* https://github.com/gfx-rs/wgpu/issues/2869